src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -785,4 +574,217 @@ public static Image getGameIcon(final Window frame) {
     }
     return img;
   }
+
+  public static void startNewTripleA(final Long maxMemory) {
```

Comment:
Begin the tacking on of stuff from TripleAProcessor. There were minimal modifications done after moving the code.